logs_ingest/main.py

```diff
@@ -102,7 +102,7 @@ def is_too_old(timestamp: str, self_monitoring: SelfMonitoring, log_part: str):
                 return True
         except Exception:
             # Not much we can do when we can't parse the timestamp
-            logging.debug(f"Failed to parse timestamp {timestamp}")
+            logging.debug(f"Failed to parse timestamp {timestamp}", "timestamp-parsing-debug")
```

Comment:
I agree with @wojtasskorcz that we shouldn't throttle debug logs but still this log should be throttled - could you change this log to exception?

Reply:
As far as I can tell by testing using integration tests, this log is more of a regular occurrence - it happens during test_main_success multiple times:
![image](https://user-images.githubusercontent.com/32064437/127972784-a179a0fc-1605-4322-a43f-c0a89eebbfc8.png)
This is how test_main_success execution looks like with this log on exception(). With debug() nothing is printed.

Reply:
Either the test should be fixed (these are definitely incorrect timestamps!) or, if these timestamps are deliberately like this, we should actually log an exception. I wouldn't expect it to be a regular occurrence in production. I agree with @mswiatkowska

Reply:
Done, thank you

Reply:
Yes, those incorrect timestamps (for several records) are used in integration test deliberately (we check if we count parsing errors correctly)